.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2
+jobs:
+  setup:
+    docker:
+      - image: node:10.13.0
+    steps:
+      - checkout
+      - run:
+          name: Initial Setup
+          command: |
+            curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+            python get-pip.py
+            curl -o- -L https://yarnpkg.com/install.sh | bash
+            export PATH="$HOME/.yarn/bin:$PATH"
+            python --version
+            python3 --version                     
+            pip --version          
+
+      - run:
+          name: Installing Dependencies
+          command: |
+            yarn install --frozen-lockfile
+            pip install --user -r requirements.txt
+
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+            - '.cache'
+            - '.eslintcache'
+
+      - run:
+          name: Running Scripts
+          command: |
+            yarn prettier --list-different
+            yarn lint
+            git config --global user.email "nihal9ns@gmail.com" # add your github email here
+            git config --global user.name "nihal9ns"    # add your github user name here
```

Comment:
What should these ones be for?

Reply:
Some tests were failing, I forgot in which spec though. But the logs were something like this : 
'no github email and username found'
'tell me who you are'

Reply:
It's probably in our gitFs tests. I'll change the user to be Renovate Bot